SofaKernel/framework/sofa/helper/vector.cpp

```diff
@@ -51,6 +52,62 @@ void SOFA_HELPER_API vector_access_failure(const void* vec, unsigned size, unsig
     assert(i < size);
 }
 
-} // namespace helper
+/// Convert the string 's' into an unsigned int. The error are reported in msg & numErrors
+/// is incremented.
+int SOFA_HELPER_API getInteger(const std::string& s, std::stringstream& msg, unsigned int& numErrors)
+{
+    const char* attrstr=s.c_str();
```

Comment:
FWIW, working directly from the `std::istream` is indeed faster than the existing parser, at least regarding your benchmark on (unsigned) integer sequences (around 15%).